miner/payload_building_test.go

```diff
@@ -125,6 +126,10 @@ func newTestWorkerBackend(t *testing.T, chainConfig *params.ChainConfig, engine
 	default:
 		t.Fatalf("unexpected consensus engine type: %T", engine)
 	}
+	if chainConfig.HoloceneTime != nil {
+		// genesis block extraData needs to be correct format
```

Comment:
I don't notice in the spec any mention of the genesis block definition of the extra data if holocene is active at genesis. This is definitely a place where a bug is waiting. For the withdrawals root feature, we did specify that it needs to be explicitly empty in the genesis state for simplicity. We likely want initial values in the extradata if holocene is active at genesis so that the values can be used during execution. This could be a good detail to add to the spec

Reply:
It's implicitly required for genesis because we state it's required for all Holocene blocks.  I can call it out explicitly in the spec.